src/tex/ms.tex

```diff
@@ -992,7 +992,7 @@ \section{OPEN-SOURCE GAUSSIAN PROCESS SOFTWARE}
 
 It is reasonably straightforward to implement a simple GP model in code, and within astrophysics it has been common for authors to implement custom GPs for their analysis.
 However, things get significantly more complicated when implementing the scalable or approximate methods discussed in the previous section.
-Similarly, it can be tedious to experiment with different kernel functions and inference methods without building a non-trivial modelling infrastructure, something that has typically been ad-hoc in astrophysics research.
+Similarly, it can be tedious to experiment with different kernel functions and inference methods without building a non-trivial modelling infrastructure, something that has typically been done ad-hoc in astrophysics research.
```

Comment:
Now that I rethink it, I believe the original formulation is much more elgant.. :>